keras_cv/layers/object_detection3d/utils_test.py

```diff
@@ -0,0 +1,57 @@
+# Copyright 2022 The KerasCV Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import tensorflow as tf
+
+from keras_cv.layers.object_detection3d.utils import PadOrTrimTo
+
+
+class PadOrTrimToTest(tf.test.TestCase):
+    """Tests for PadOrTrimTo, branched from https://github.com/tensorflow/lingvo/blob/master/lingvo/core/py_utils_test.py."""
+
+    def test2DConstantShapePad(self):
```

Comment:
nit: lowercase_underscore naming for these test methods too

Reply:
done